.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: java
+
+jdk:
+  - openjdk7
```

Comment:
OpenJDK 7  is what is currently on was-downloader, and I had the impression that was going to be the java installed on the new VMs.